### PR TITLE
[TASK] Add section on keyboard commands

### LINKED ISSUE #28: Add useful keyboard shortcuts?
see

* https://www.typo3manual.com/typo3-basics/tips-and-functions/short-commands/

Also, Shift-reload invalidates TYPO3 cache as well and refreshes it, see

* https://github.com/TYPO3/TYPO3.CMS/blob/36994dd3cc28ecca23b42a62d33ae08b54261c3b/typo3/sysext/frontend/Classes/Controller/TypoScriptFrontendController.php#L2465-L2471

* https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/Setup/Config/Index.html#sendcacheheaders

### PR DESCRIPTION
**Pull request by tomwarwick**
Resolves #28

### CODE REVIEW

File: Documentation/HelpInside/KeyboardCommands/Index.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Keyboard Commands
+=================
+
+Navigating the Page Tree using a keyboard
+=========================================
+
+It is possible to navigate the page tree using only a keyboard and it adheres to
+the guidelines set out in `WAI-ARIA Authoring Practices 1.1. <https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-22>`__
+
+* `Home` moves the focus to the first item in the page tree.
```
Comment (sypets) [inline Documentation/HelpInside/KeyboardCommands/Index.rst:15]: For keys, I would recommend the use of the :kbd: textrole, see https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/InlineCode.html#using-text-roles

e.g. 

```
:kbd:`Home`
```

**Review by sypets — COMMENTED** [review]
This looks very nice. I already pointed out a problem with a menu. I cannot proofread if the keys are correct.

File: Documentation/HelpInside/KeyboardCommands/Index.rst (hunk 1 above)
Comment (tomwarwick) [reply Documentation/HelpInside/KeyboardCommands/Index.rst]: Resolved - thank you.

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
To me it looks fine so far. Are you still working on this or why is it in draft?

**Pull request merged**